package.json

```diff
@@ -63,16 +63,17 @@
     "@typescript-eslint/eslint-plugin": "^5.33.0",
     "@typescript-eslint/parser": "^5.33.0",
     "copyfiles": "^2.4.1",
-    "electron": "^26.1.0",
+    "electron": "^26.4.0",
     "eslint": "^8.22.0",
     "eslint-plugin-import": "^2.26.0",
     "jest": "^28.1.3",
     "jest-image-snapshot": "^5.1.0",
     "jest-mock-extended": "^3.0.4",
+    "langchain": "0.0.125",
```

Comment:
as long as the plugin is in the core repository we should add langchain to "dependencies" as it is also needed at production